src/lib.rs

```diff
@@ -41,24 +63,55 @@ extern crate proc_macro2;
 
 use proc_macro::TokenStream;
 use proc_macro2::TokenStream as TokenStream2;
-use syn::{DataStruct, DeriveInput, Field};
+use syn::{Attribute, DataStruct, DeriveInput, Field, Ident, Lit, Meta};
 
 mod generate;
 use generate::{GenMode, GenParams};
 
-#[proc_macro_derive(Getters, attributes(get))]
+fn attr_name(attr: &Attribute) -> Option<Ident> {
+    attr.interpret_meta().map(|v| v.name())
+}
+
+/// Some users want legacy/compatability.
+/// (Getters are often prefixed with `get_`)
+fn has_with_prefix_set(f: &Field) -> bool {
```

Comment:
I think this function name makes sense, but unless you know what is happening it reads a bit like word salad :-). Maybe `has_prefix_attr` or something similar?